test/sharness/t0184-http-proxy-over-p2p.sh

```diff
@@ -167,7 +167,7 @@ test_expect_success 'setup p2p listener on the receiver' '
 '
 
 test_expect_success 'setup environment' '
-    RECEIVER_ID="$(iptb attr get 1 id)"
```

Comment:
@petar does this test fail if we only use base36 representation for the RECEIVER_ID_CIDv1 below? I thought the gateway redirect would work here. If not then we might need a minor modification to the gateway code.